src/tools/compiletest/src/runtest.rs

```diff
@@ -2470,7 +2470,7 @@ impl<'test> TestCx<'test> {
             }
             CoverageMap => {
                 rustc.arg("-Cinstrument-coverage");
-                // These tests only compile to MIR, so they don't need the
+                // These tests only compile to LLVM IR, so they don't need the
```

Comment:
This is a mostly-unrelated comment fix that I happened to notice at around the same time.